signbank/dictionary/csv_import.py

```diff
@@ -650,11 +650,15 @@ def confirm_import_qualtrics_csv(request):
     validation_records_added = []
     validation_records = []
     missing_gloss_pk_question_pairs = {}
+    bulk_tagged_items = []
 
     if "validation_records" and "question_numbers" and "question_gloss_map" in request.session:
         # Retrieve glosses
         gloss_pk_list = request.session["question_gloss_map"].values()
         gloss_dict = Gloss.objects.in_bulk(gloss_pk_list)
+        gloss_content_type = ContentType.objects.get_for_model(Gloss)
+        check_result_tag = Tag.objects.get(name=settings.TAG_VALIDATION_CHECK_RESULTS)
```

Comment:
Should we potentially use `get_or_create` here?

Reply:
This particular tag has been created in a migration